extensions/devservices/oracle/src/main/java/io/quarkus/devservices/oracle/deployment/OracleDatasourceServiceConfigurator.java

```diff
@@ -36,7 +36,7 @@ public RunningDevServicesDatasource composeRunningService(ContainerAddress conta
     }
 
     public String getJdbcUrl(ContainerAddress containerAddress, String databaseName) {
-        return "jdbc:%s://@%s:%d/%s%s".formatted(
+        return "jdbc:%s:@%s:%d/%s%s".formatted(
```

Comment:
What bugs me is that I fixed it at some point here: https://github.com/quarkusio/quarkus/commit/45d4db21c224cc9c9a0811b130e1de94fd51cb00 .

So either it got lost in translation or it was changed for a reason.